src/vmm/src/device_manager/mmio.rs

```diff
@@ -204,6 +202,8 @@ impl MMIODeviceManager {
         device_id: &str,
     ) -> Result<()> {
         self.register_device_resources(vm, transport_device, cmdline, device_id, TYPE_BLOCK)?;
+        // Temporarly use this hashmap. It is used to retrieve the Block object
```

Comment:
nit: add a `TODO` or `FIXME` here to make it clear the code is in flux.

Reply:
There is an incoming commit from Alex which will fix it soon.